### PR TITLE
updates PR template

### PR DESCRIPTION
**Pull request by aarongranick-okta**
## Description:

Updates PR template to include more detail about testing

### Issue:

- [OKTA-514517](https://oktainc.atlassian.net/browse/OKTA-514517)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -5,23 +5,23 @@
...
 - [ ] Have you verified the basic functionality for this change?
-
-- [ ] Added unit tests?
-
-- [ ] Added e2e tests
-
+- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
+- [ ] Does this PR include changes which are untested or may affect areas with [known gaps in automated testing](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2630719564/Automation+Gaps)? (If yes, provide test plan below)
 - [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
+- [ ] Did you verify the change by running downstream monolith artifacts? (Provide link to build below)
```
Comment (lesterchoi-okta) [inline .github/pull_request_template.md:11]: Do we have a link with instructions on how to run a d16t? Not sure if every contributor knows

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -5,23 +5,23 @@
...
+- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)
 
-- Did you verify the change by running downstream monolith artifacts? (YES | NO | UNSURE)
-
-### Screenshot/Video:
+### Issue:
 
+- [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)
```
Comment (lesterchoi-okta) [inline .github/pull_request_template.md:16]: nit: while you're here, [okta-core](https://github.com/okta/okta-core/blob/master/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L40) just has 

```suggestion
- OKTA-XXXXXX
```

and it appears to linkify correctly

**Review by lesterchoi-okta — APPROVED** [review]
LGTM, would nudge @robertdamphousse-okta and someone on QA to double-check

File: .github/pull_request_template.md (hunk 1 above)
Comment (lesterchoi-okta) [reply .github/pull_request_template.md:11]: Just got this question from an eng who didn't know how to trigger the d16t build.
Can we paste instructions from here https://github.com/okta/okta-core/pull/68408 into a wiki and link it?

Comment (aarongranick-okta) [reply .github/pull_request_template.md]: We have instructions here: https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t)
Adding to PR template

**Pull request merged**